src/presentation/test/integration/registration/general-partner-choice.test.ts

```diff
@@ -14,6 +14,7 @@ import {
   APPLICATION_CACHE_KEY,
   APPLICATION_CACHE_KEY_PREFIX_REGISTRATION,
 } from "../../../../config/constants";
+import LimitedPartnershipBuilder from "../../builder/LimitedPartnershipBuilder";
 
 describe("General Partner Choice Page", () => {
   beforeEach(() => {
```

Comment:
you forgot to reset data in registrationGateway before each testt

Reply:
data reset added

Reply:
reset data added